content/network/drivers/overlay.md

```diff
@@ -10,62 +10,37 @@ aliases:
+encryption is enabled. Docker transparently handles routing of each packet to
+and from the correct Docker daemon host and the correct destination container.
 
 You can create user-defined `overlay` networks using `docker network create`,
 in the same way that you can create user-defined `bridge` networks. Services
 or containers can be connected to more than one network at a time. Services or
-containers can only communicate across networks they are each connected to.
-
-Although you can connect both swarm services and standalone containers to an
-overlay network, the default behaviors and configuration concerns are different.
-For that reason, the rest of this topic is divided into operations that apply to
-all overlay networks, those that apply to swarm service networks, and those that
-apply to overlay networks used by standalone containers.
-
-## Operations for all overlay networks
-
-### Create an overlay network
-
-Take note of the following prerequisites:
+containers can only communicate across networks they're each connected to.
 
-- Firewall rules for Docker daemons using overlay networks.
+Overlay networks are often used to create a connection between Swarm services,
+but you can also use it to connect standalone containers running on different
+hosts. When using standalone containers, it's still recommended that you use
+Swarm mode to establish a connection between the hosts. It's a lot easier set
+up and maintain, compared to the alternative of running an external key-value
+database for service discovery.
 
-  You need the following ports open to traffic to and from each Docker host participating on an overlay network:
-  - TCP port 2377 for cluster management communications
-  - TCP and UDP port 7946 for communication among nodes
-  - UDP port 4789 for overlay network traffic
+This page describes overlay networks in general, and when used with standalone
+containers. For information about overlay for Swarm services, see
+[Manage Swarm service networks](../../engine/swarm/networking.md).
 
-- Before you can create an overlay network, you need to either initialize your
-  Docker daemon as a swarm manager using `docker swarm init` or join it to an
-  existing swarm using `docker swarm join`. Either of these creates the default
-  `ingress` overlay network which swarm services use by default. You need
-  to do this even if you never plan to use swarm services. Afterward, you can
-  create additional user-defined overlay networks.
+## Create an overlay network
 
-To create an overlay network for use with swarm services, use a command like
-the following:
+Before you start, you need the following ports open to traffic to and from each
+Docker host participating in an overlay network:
 
-```console
-$ docker network create -d overlay my-overlay
-```
+- TCP port 2377 for cluster management communications
+- TCP and UDP port 7946 for communication among nodes
+- UDP port 4789 for overlay network traffic
```

Comment:
2377/tcp is the default Swarm control plane port, and is configurable (`docker swarm join --listen-addr`). 4789/udp is the default overlay traffic port, and is also configurable (`docker swarm init --data-path-port`). Port 7946 is inexplicably hardcoded, though.